src/main/webapp/index.html

```diff
@@ -6,10 +6,21 @@
 </head>
 <body>
   <p></p>
+  <a></a>
   <script>
       fetch('/hello', {method: 'GET'}).then(response => response.text()).then((text) => {
           document.querySelector('p').innerText = text;
```

Comment:
Same comment as below. Please add an id to the `<p>` element and use it to select the element.